src/locales/ko/fight-ui-handler.ts

```diff
@@ -4,4 +4,6 @@ export const fightUiHandler: SimpleTranslationEntries = {
   "pp": "PP",
   "power": "위력",
   "accuracy": "명중률",
+  "abilityFlyInText": " {{pokemonName}}'s {{passive}}{{abilityName}}",
+  "passive": "Passive ", // The space at the end is important
```

Comment:
```suggestion
  "abilityFlyInText": " {{pokemonName}}의 {{passive}}{{abilityName}}",
  "passive": "패시브 ", // The space at the end is important
```
Korean translation added